dev/com.ibm.websphere.appserver.features/visibility/public/restfulWS-3.1/io.openliberty.restfulWS-3.1.feature

```diff
@@ -9,6 +9,7 @@ IBM-ShortName: restfulWS-3.1
 WLP-AlsoKnownAs: jaxrs-3.1
 Subsystem-Name: Jakarta RESTful Web Services 3.1
 -features=\
+  io.openliberty.jakarta.concurrency-3.0, \
```

Comment:
```suggestion
```

This is unnecessary since it is in restfulWSClient-3.1

Reply:
Agreed / done.